docs/embedded/getting-started/register-api-documentation.md

```diff
@@ -69,6 +70,16 @@ If successful, this method returns a `200 OK` response code and the container ty
 
 ### Register the container type in a consuming tenant
 
+To interact with containers in a consuming tenant, the owning tenant must first be granted permissions by the consuming tenant admin.  This step is crucial as it controls permissions and prevents access denied errors when invoking other APIs.
```

Comment:
Move this to the top of the page. This is not part of the example. Assume no one reads examples and they should still be successful. This is a requirement for readers to be successful.